operator/pkg/ingress/annotations/annotations.go

```diff
@@ -30,6 +31,7 @@ const (
 	InsecureNodePortAnnotationAlias = annotation.Prefix + ".ingress" + "/insecure-node-port"
 	SecureNodePortAnnotationAlias   = annotation.Prefix + ".ingress" + "/secure-node-port"
 	TLSPassthroughAnnotationAlias   = annotation.Prefix + ".ingress" + "/tls-passthrough"
+	EnforceHTTPSAnnotationAlias     = annotation.Prefix + ".ingress" + "/enforce-https"
```

Comment:
do (and should) we support the legacy/fallback/alias annotation (`io.cilium.ingress/...`) for new annotations too?

IMO we shouldn't.

Reply:
I've been adding them in just in case, but I agree, let's stop doing that. I'll remove.